### DISCUSSION TITLE
Translate `Back to top`

### DISCUSSION THREAD

**Discussion by kamieniarz** [squidfunk/mkdocs-material#5267] [Q&A] [answered]
Is it possible to translate `Back to top` if I launch mkdocs with mkdocs serve? So basically I only have mkdocs.yml file and docs dir so I don't have source files to edit maunally stuff like that

**Comment by kamilkrzyskow** [comment, marked as answer]
Hi, you can create a custom translation without having to modify the source directly.

https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#custom-translations

To change the Back to Top text override the "top" key
https://github.com/squidfunk/mkdocs-material/blob/master/src/partials/languages/en.html#L78

Also consider adding your language or give suggestions on how to improve the current languages to the repository.✌️
